website/src/plugins/test-resourcecs/mock-release-data.ts

```diff
@@ -0,0 +1,1718 @@
+export const mockReleaseData = {
```

Comment:
question: if there is no type, can't we just have this in a json file please?

Reply:
I added the type, let me know if you still prefer a json file